release-notes/10.0/preview/preview2/aspnetcore.md

```diff
@@ -15,6 +22,206 @@ ASP.NET Core updates in .NET 10 Preview 2:
 - [Discussion](https://aka.ms/dotnet/10/preview2)
 - [Release notes](README.md)
 
-## Feature
+## Reconnection UI component added to the Blazor Web App project template
 
-This is about the feature
+The Blazor Web App project template now includes a `ReconnectModal` component, including collocated stylesheet and JavaScript files, for improved developer control over the reconnection UI when the client loses the WebSocket connection to the server. The component doesn't insert styles programmatically, so it doesn't violate stricter Content Security Policy (CSP) settings for the `style-src` policy. A default reconnection UI is still used as fallback when the app doesn't define its own reconnection UI.
```

Comment:
```suggestion
The Blazor Web App project template now includes a `ReconnectModal` component, including collocated stylesheet and JavaScript files, for improved developer control over the reconnection UI when the client loses the WebSocket connection to the server. The component doesn't insert styles programmatically, ensuring compliance with stricter Content Security Policy (CSP) settings for the `style-src` policy. A default reconnection UI is still used as fallback when the app doesn't define its own reconnection UI.
```

Reply:
I'm including this update on https://github.com/dotnet/AspNetCore.Docs/pull/34966 for the ASP.NET Core *Release Notes* article.